lib/internal/fs/streams.js

```diff
@@ -395,6 +398,18 @@ WriteStream.prototype.close = function(cb) {
 // There is no shutdown() for files.
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
+WriteStream.prototype.fsync = function(callback) {
```

Comment:
I manual merge `fs.fsync` and `fs.fsyncSync` to `WriteStream.prototype.fsync`

Is this right?

Reply:
or use directly `fs.fsync` and `fs.fsyncSync`